components/user-store/org.wso2.carbon.identity.user.store.configuration.ui/src/main/resources/web/userstore_config/userstore-config.jsp

```diff
@@ -113,7 +111,7 @@
 
 
     if (selectedClassApplied == null || selectedClassApplied.trim().length() == 0) {
-        selectedClassApplied = UserStoreUIConstants.RWLDAP_USERSTORE_MANAGER;
+        selectedClassApplied = UserStoreUIConstants.UROLDAP_USERSTORE_MANAGER;
```

Comment:
Shouldn't this be Unique Read Write LDAP?

Reply:
The current default is the Unique Read Only LDAP (i.e. first option in the dropdown box).

Reply:
Changed to UniqueIDReadWriteLDAP in dc3b8ad94f80cf58e55d175f705f4c15863becf7